droid-hal-device.inc

```diff
@@ -596,7 +596,8 @@ popd 1>/dev/null
 
 # Add our config into the default android-config.h
 echo Making new $HDRS/android-config.h
-sed '/CONFIG GOES HERE/,$d' $HDRS/android-config.h > $HDRS/android-config.h.new
+TOKEN_LINE=$(sed -n '/CONFIG GOES HERE/=' $HDRS/android-config.h)
```

Comment:
Uppercase variables are usually for shell's use, or conventionally for global exports. Script-internal (local) variables best look when in lowercase.

Reply:
> That is true for shell scripts but in this case all variables are mostly uppercase and wanted ti follow the rest of the specfile.

That doesn't mean we shouldn't strive to improve our coding style:)